go.mod

```diff
@@ -3,6 +3,12 @@ module github.com/layer5io/meshery-operator
 go 1.13
 
 require (
+	github.com/Azure/go-autorest/autorest/adal v0.9.0 // indirect
+	github.com/Sirupsen/logrus v1.6.0 // indirect
```

Comment:
Curious, did you have to handcraft this or did this come about after a the `go mod replace` you mentioned?

Reply:
Yup had to handcraft this.